docs/content/integrations/dbt-cloud.mdx

```diff
@@ -28,78 +46,74 @@ width={1055}
 height={673}
 />
 
-The `auth_token` can also be found or generated in the dbt Cloud console. It's recommended that you use a [Service account token](https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/service-tokens) for this purpose, and that you store this value in an environment variable, rather than hardcoding its value in your codebase.
+---
 
-Putting it all together, you'll get the following:
+## Step 1: Connecting to dbt Cloud
 
-```python startafter=start_dbt_cloud_job endbefore=end_dbt_cloud_job file=/integrations/dbt/dbt_cloud.py dedent=4
-from dagster import job
-from dagster_dbt import dbt_cloud_resource, dbt_cloud_run_op
+The first step in using dbt Cloud with Dagster is to tell Dagster how to connect to your dbt Cloud instance using an dbt Cloud [resource](/concepts/resources). This resource contains information on where the dbt Cloud instance is located and any credentials needed to access it.
 
-# configure an operation to run the specific job
-run_dbt_nightly_sync = dbt_cloud_run_op.configured(
-    {"job_id": 33333}, name="run_dbt_nightly_sync"
-)
+```python startafter=start_define_dbt_cloud_instance endbefore=end_define_dbt_cloud_instance file=/integrations/dbt/dbt_cloud.py dedent=4
+from dagster_dbt import dbt_cloud_resource
 
-# configure a resource to connect to your dbt Cloud instance
-my_dbt_cloud_resource = dbt_cloud_resource.configured(
-    {"auth_token": {"env": "DBT_CLOUD_AUTH_TOKEN"}, "account_id": 11111}
+dbt_cloud_instance = dbt_cloud_resource.configured(
+    {
+        "auth_token": {"env": "DBT_CLOUD_API_TOKEN"},
+        "account_id": {"env": "DBT_CLOUD_ACCOUNT_ID"},
+    }
 )
-
-# create a job that uses your op and resource
-@job(resource_defs={"dbt_cloud": my_dbt_cloud_resource})
-def my_dbt_cloud_job():
-    run_dbt_nightly_sync()
 ```
 
----
+## Step 2: Loading dbt Cloud models as assets
 
-## Running a dbt Cloud job after another op completes
+In this step, you'll load the dbt Cloud models managed by a dbt Cloud job into Dagster as assets. For context, a dbt Cloud job defines set of commands to run for a dbt Cloud project.
```

Comment:
I think its worth clarifying what "managed by a dbt Cloud job" means. For instance, are all the models imported in the dbt job only "manages" a subset?